pkgs/tools/graphics/piglit/default.nix

```diff
@@ -0,0 +1,76 @@
+{ stdenv
+, fetchgit
+, lib
+, cmake
+, freeglut
+, libGL
+, libGLU
+, libglvnd
+, makeWrapper
+, ninja
+, pkg-config
+, python3Packages
+, waffle
+, wayland
+, libX11
+, libXrender
+, libxcb
+, libxkbcommon
+}:
+
+stdenv.mkDerivation rec {
+  pname = "piglit";
+  version = "unstable-2020-10-28";
+
+  src = fetchgit {
```

Comment:
`rev` should be specified.

Reply:
The same, consider using `fetchFromGitLab`:
```nix
  src = fetchFromGitLab {
    domain = "gitlab.freedesktop.org";
    owner = "mesa";
    repo = "piglit";
    rev = "59e695c16fdcdd4ea4f16365f0e397a93cef7b80";
    sha256 = "03yszpvjx6bgbkfgw2y9lajq498qmrirf24hdv9csxbg4zckw7ck";
  };
```

Reply:
whoops, thanks for your review!